packages/labs/ssr/src/lib/lit-element-renderer.ts

```diff
@@ -85,6 +86,22 @@ export class LitElementRenderer extends ElementRenderer {
   }
 
   override *renderShadow(renderInfo: RenderInfo): RenderResult {
+    const serverControllers = Array.from<Partial<ServerController>>(
+      getControllers(this.element) ?? []
+    )
+      .map((c: Partial<ServerController>) => c.serverUpdateComplete)
+      .filter((p: Promise<unknown> | undefined) => !!p);
+    if (serverControllers?.length > 0) {
+      const continuation = Promise.allSettled(serverControllers).then((_) =>
```

Comment:
Hm, this means that if a serverController's updateComplete promise rejects, we will silently ignore that error, which isn't good. I think we should at least console.error log it.

Reply:
Alternatively, I can change to `all` so it rejects.

Reply:
Changed to Promise.all. Will also add a test for this case.

Reply:
Turns out, it's not easy to add a test for this case, as it's a catastrophic failure. But that is better than a silent failure, and matches more closely how we fail in other areas.